src/Lykke.Service.CryptoIndex.Domain/Models/MarketCap.cs

```diff
@@ -10,7 +10,7 @@ public class MarketCap
 
         public MarketCap(decimal value, string asset)
         {
-            if (value == default(decimal)) throw new ArgumentOutOfRangeException(nameof(value));
+            if (value < 0) throw new ArgumentOutOfRangeException(nameof(value));
             if (string.IsNullOrWhiteSpace(asset)) throw new ArgumentOutOfRangeException(nameof(asset));
```

Comment:
ArgumentOutOfRangeException does not mean that the asset is empty, what about ArgumentNullException|ArgumentException?